xbmc/settings/SettingsValueXmlSerializer.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (C) 2019 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "SettingsValueXmlSerializer.h"
+
+#include "settings/lib/SettingDefinitions.h"
+#include "settings/lib/SettingsManager.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
What exactly is wrong here? `SettingDefinitions.h` should come before `SettingsManager.h` right?

Reply:
Yes that's right, but `settings/lib/SettingSection.h` before `settings/lib/SettingsManager.h` (case sensitive sorting).

Reply:
Ah I must have been blind. I only looked at the two lines above. Fixed.